.github/workflows/tests.yaml

```diff
@@ -18,12 +18,10 @@ jobs:
 
     runs-on: ${{ matrix.os }}
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
```

Comment:
Hello! Thanks for this! Question, why the removal of `fetch-depth`? Thanks!

Reply:
Removing the fetch-depth would just do the HEAD build, would it disrupt anything?

Reply:
Ah, I see